src/unisparks/vest.cpp

```diff
@@ -48,7 +48,22 @@ void vestLoop(void) {
 #endif // ESP32_BLE
 
   player.render();
-  mainVestController->showLeds(getBrightness());
+  uint32_t br = getBrightness();        // May be reduced if this exceeds our power budget with the current pattern
+
+#if MAX_MW
```

Comment:
nit: the meaning of `MAX_MW` isn't immediately obvious, how about something self-descriptive like `MAX_MILLIWATTS`?